projects/go-lib/src/lib/components/go-icon/go-icon.component.ts

```diff
@@ -1,10 +1,25 @@
-import { Component, Input } from '@angular/core';
+import { Component, Input, OnInit } from '@angular/core';
 
 @Component({
   selector: 'go-icon',
-  templateUrl: 'go-icon.component.html',
-  styleUrls: ['go-icon.component.scss']
```

Comment:
We need the contents of this sass file. It allows us to properly override the font-size of the icon

Reply:
We adjusted the styling in `gosheets` to fix the issue we were overriding in this file, thus we now do not need it.